proto/dependencies.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+# Copyright 2017 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+echo Downloading proto files.
+
+mkdir -p k8s.io/apimachinery/pkg/api/resource
+mkdir -p k8s.io/apimachinery/pkg/apis/meta/v1
+mkdir -p k8s.io/apimachinery/pkg/util/intstr
+mkdir -p k8s.io/apimachinery/pkg/runtime/schema
+mkdir -p k8s.io/apis/meta/v1
```

Comment:
are these all generated.proto files we need?

Reply:
For the basic V1 API. Doesn't cover any of the other api groups, we can add them later?